### PR TITLE
Hide products with visibility none

### PR DESCRIPTION
**Pull request by SebBareyre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Hide products with visibility none
| Type?         | bug fix
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | .
| How to test?  | Consult a product with suggested products by cross selling module. Change one of these products and set it in visibility none. this PR will hide this product.

### CODE REVIEW

File: ps_crossselling.php [hunk 1]
```diff
@@ -299,6 +299,7 @@ protected function getOrderProducts(array $productIds = [])
                 WHERE od.id_order IN (' . $list . ')
                 AND od.product_id NOT IN (' . $list_product_ids . ')
                 AND product_shop.active = 1
+                AND product_shop.`visibility` != \'none\'
```
Comment (NeOMakinG) [inline ps_crossselling.php:302]: I'm surprised the visibility field is a string, waiting for @PrestaShop/prestashop-maintainers as I'm not comfortable with the PHP/DB stuff

Comment (Progi1984) [reply ps_crossselling.php:302]: Yes, it's normal :

https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Product.php#L245-L246

**Review by Progi1984 — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
Hello @SebBareyre 

Could you please create an issue and assign it to this PR, or assign an existing issue to this PR!

Thanks in advance :pray:

**Comment by matks** [comment]
Dear @SebBareyre could you please create a bug report to explain the issue you're solving ? 😉  

I know the change looks obvious but bug reports are great help for the project and the QA team.

If we have the clear list of steps to reproduce the issue, anybody can reproduce the issue and anybody can test the bugfix submitted to fix it. So it's transparent and a mark of trust. 👍 

Also if we can classify and then it will be easier in the future to link it to other issues, to have statistics about how many bugs solved in one area, it is even easier for people that encounter the same issue to search for it using labels and fix the solution.

**Review by lmeyer1 — APPROVED** [review]
Thanks for your contribution.

**Comment by matks** [comment]
> Dear @SebBareyre could you please create a bug report to explain the issue you're solving ? 😉
> 
> I know the change looks obvious but bug reports are great help for the project and the QA team.
> 
> If we have the clear list of steps to reproduce the issue, anybody can reproduce the issue and anybody can test the bugfix submitted to fix it. So it's transparent and a mark of trust. 👍
> 
> Also if we can classify and then it will be easier in the future to link it to other issues, to have statistics about how many bugs solved in one area, it is even easier for people that encounter the same issue to search for it using labels and fix the solution.

It's me again, @SebBareyre could you please create a bug report to explain the issue you're solving ? 😉

**Comment by matks** [comment]
> It's me again, @SebBareyre could you please create a bug report to explain the issue you're solving

Since we had no news from you for many months, I will follow [our guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/#stale-pull-requests) and I'll close this pull request as it seems stale. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**